core/arch/opnd.h

```diff
@@ -2351,18 +2435,52 @@ DR_API
  * Assumes that \p reg is a DR_REG_ constant.
  * Returns true iff it refers to an xmm (128-bit SSE/SSE2) register
  * or a ymm (256-bit multimedia) register.
+ * \deprecated Prefer reg_is_strictly_xmm().
```

Comment:
That does not do the same thing.  You mean " || reg_is_strictly_ymm()".

Reply:
Done.